fancy-explosions/BangazonWorkforce
# Training Programs List and Create

# Description

This pull request is for the added ability to view a list of all active training programs and the ability to create a new training program.

Fixes Issue # (`https://github.com/fancy-explosions/BangazonWorkforce/issues/9`)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. Git checkout bm-TrainingPrograms-1
2. Review code.
3. Run app
4. Check check for training programs list functionality.
5. Check check for training programs create functionality.
6. Make sure the old controllers have been moved out of main project and into root directory in a ReferenceController folder.

# Checklist:
